docs/reference.md

```diff
@@ -1873,9 +1873,9 @@ Key | Type | Required |
 [body](#body) | Object/String | No |
 [headers](#headers) | String | No |
 
-_nb: For multipart/form-data content type requests, the body should be provided as a key/value object. For other content types, it can be provided as String in JSON format._
+_nb: For multipart/form-data content type requests, the body should be provided as a key/value object. For other content types, String in JSON format is also supported._
```

Comment:
if we support JSON now why don't we say so?

Reply:
Because JSON string is not supported for multipart/form-data. 
I updated this in "body" description to say that it can be provided as JSON or key/value object, but only key/value when using mulitpart/form.